src/Sarif/Schemata/sarif-2.1.0-rtm.5.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
>$schema [](http://example.com/codeflow?start=3&length=7)

Sorry for the late-breaking comment! You need to produce an rtm.6 version of this file, we can't modify rtm.5, it's an artifact of the standards process. Ask Chris Meyer for information about how to update everything to use a new flavor of schema.

Sorry again for the delay giving you this advice.